llvm-tools/src/main/scala/scala/scalanative/llvm/LLVM.scala

```diff
@@ -0,0 +1,158 @@
+package scala.scalanative
+package llvm
+
+import java.io.File
+
+import scala.sys.process.{Process, ProcessLogger}
+import scala.util.Try
+
+import System.{lineSeparator => nl}
+
+object LLVM {
```

Comment:
Currently the separation seems to be quite awkward, given that some parts like `unpackNativelib` are still part of the sbt plugin.